src/redux/store.js

```diff
@@ -1,6 +1,10 @@
-import { createStore } from 'redux';
+import { createStore, applyMiddleware } from 'redux';
+import logger from 'redux-logger';
 import rootReducer from './rootReducer';
 
-const store = createStore(rootReducer);
+const store = createStore(
+    rootReducer,
+    applyMiddleware(logger),
+);
```

Comment:
- [ ] You should import and use `combineReducers` to create a store instead of doing so in the `rootReducer` file, please check the projects requirements.

Reply:
sorry, I made it in that way following the SOLID principle called Single Responsibility Principle. However, in just a while I'll fit the requirement.